ddp-workspace/projects/ddp-sdk/src/lib/directives/sticky-scroll.directive.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { StickyScrollDirective } from './sticky-scroll.directive';
+import { TestBed } from '@angular/core/testing';
+import { Component, DebugElement } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { WindowRef } from 'ddp-sdk';
+
+const notRecognizableTopSpace = 73;
+const containerHeight = 50;
+// less then container height in order to see the scroll
+const windowHeight = containerHeight - 10 + notRecognizableTopSpace;
+
+function getTestComponentTemplateWithStyles({hostWidth = '100px', padding = '0'}: {hostWidth?: string; padding?: string} = {}): string {
```

Comment:
#optional
`({hostWidth = '100px', padding = '0'}: {hostWidth?: string; padding?: string} = {})`
vs 
`(hostWidth: string = '100px', padding?: string = '0')`

Reply:
it's much better, thanks! done